internal/config/5_real_config.go

```diff
@@ -13,7 +13,6 @@ import (
 func init() {
 	if clowder.IsClowderEnabled() {
 		cfg := clowder.LoadedConfig
-		viper.Set("restEndpoints.sources.appTypeId", 11)
```

Comment:
We have the same below, and also in the `5_test_config.go`. Just saying, I am not entirely sure about this, so for non-clowder environment we say the type id is 11? That is still weird.